src/Adapter/MSTest.TestAdapter/Execution/TestExecutionManager.cs

```diff
@@ -372,20 +372,7 @@ private async Task ExecuteTestsInSourceAsync(IEnumerable<TestCase> tests, IRunCo
 
         _testRunCancellationToken?.ThrowIfCancellationRequested();
 
-        TestAssemblySettingsProvider? sourceSettingsProvider = null;
-
-        try
-        {
-            sourceSettingsProvider = isolationHost.CreateInstanceForType(
```

Comment:
We previously were creating an instance of `TestAssemblySettingsProvider` in a separate AppDomain, but we did "nothing" with it other than a null check. Then we simply call `TestAssemblySettingsProvider.GetSettings`